source/bson-corpus/tests/code.json

```diff
@@ -20,48 +20,48 @@
         },
         {
             "description": "two-byte UTF-8 (\u00e9)",
-            "canonical_bson": "190000000261000D000000C3A9C3A9C3A9C3A9C3A9C3A90000",
-            "canonical_extjson": "{\"a\" : \"\\u00e9\\u00e9\\u00e9\\u00e9\\u00e9\\u00e9\"}"
+            "canonical_bson": "190000000D61000D000000C3A9C3A9C3A9C3A9C3A9C3A90000",
```

Comment:
Note: the only diff to these byte sequences was changing `02` to `0D`. Thankfully, code w/o scope and strings have the same structure.

Likewise for the `decodeErrors` tests below.